src/client/screens/Product.js

```diff
@@ -1,41 +1,30 @@
 import React, { useState } from 'react';
 import { useLocation } from 'react-router-dom';
-import { connect } from 'react-redux';
-import { addProduct } from '../redux/actions/basket'
-import '../styles/product.css';
 import { Grid, Button, Snackbar } from '@material-ui/core';
+import Alert from '@material-ui/lab/Alert';
```

Comment:
@VRatick 
Dont use material-ui in this project.

Reply:
Fixed